site/content/en/blog/releases/1.27.0-rc.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "1.27.0 - Release Candidate"
+linkTitle: "1.27.0-rc"
+date: "2022-10-19"
+---
+This is the 1.27.0-rc release of Agones.
+
+In this release Agones upgrades to Go 1.19 (from 1.17!), so we can take advantage of all the [improvements therein](https://tip.golang.org/doc/go1.19).
+
+This release has lots of important changes including:
+- Enable image streaming everywhere by default on GKE
+- Provide more flexibility on utilising the cluster autoscaler with graceful termination.
+- Reduced logging verbosity by wrapping an error into function.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.27.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
+
+- Allow cluster autoscaler to scale down game server pods [\#2747](https://github.com/googleforgames/agones/issues/2747)
+- \[GKE\] - Should we enable image streaming everywhere? [\#2746](https://github.com/googleforgames/agones/issues/2746)
+- Support Agones on ARM systems [\#2216](https://github.com/googleforgames/agones/issues/2216)
+- Update example containers to fix security vulnerabilities [\#1154](https://github.com/googleforgames/agones/issues/1154)
+
+Documentation: https://development.agones.dev/site/
+
+
+See <a href="https://github.com/googleforgames/agones/blob/release-1.27.0-rc/CHANGELOG.md" data-proofer-ignore>CHANGELOG</a> for more details on changes.
+
+Images available with this release:
+
+- [gcr.io/agones-images/agones-controller:1.27.0-rc](https://gcr.io/agones-images/agones-controller:1.27.0-rc)
+- [gcr.io/agones-images/agones-sdk:1.27.0-rc](https://gcr.io/agones-images/agones-sdk:1.27.0-rc)
+- [gcr.io/agones-images/agones-ping:1.27.0-rc](https://gcr.io/agones-images/agones-ping:1.27.0-rc)
+- [gcr.io/agones-images/agones-allocator:{1.27.0-rc](https://gcr.io/agones-images/agones-allocator:1.27.0-rc)
```

Comment:
There is an extra `{` left in this line. It should be `gcr.io/agones-images/agones-allocator:1.27.0-rc`